.github/workflows/build-dumb-pypi.yml

```diff
@@ -159,7 +159,8 @@ jobs:
       env:
         _ANSIBLE_SDIST_FROM_MAKEFILE: 1
       run: |
-        make sdist_check
+        packaging/release.py build
+        packaging/release.py test
         python -m setup sdist --dist-dir=../dist
```

Comment:
Invoking `setup.py` directly is deprecated in `setuptools` FYI. Plus, `packaging/release.py build` has already built the sdist by this point, I think.

Reply:
The result still needs to be copied to `../dist/` I guess.

Reply:
I don't recommend using the `packaging/release.py` script. It's intended for use by the release managers when releasing ansible-core. I don't see any reason why `python -m build` can't be used directly instead.

Reply:
I mainly used `packing/release.py` because it was mentioned in https://github.com/ansible/ansible/pull/80347. I've switched it to `python -m build` in df06fcdd69d62d1025d2f89f7872f871d1abb76c, now it works fine.